openslides/motions/models.py

```diff
@@ -378,56 +389,37 @@ def get_last_version(self):
         except IndexError:
             return self.get_new_version()
 
-    @property
-    def submitters(self):
-        return sorted([object.person for object in self.submitter.all()],
-                      key=lambda person: person.sort_name)
-
-    def is_submitter(self, person):
-        """Return True, if person is a submitter of this motion. Else: False."""
-        return self.submitter.filter(person=person.pk).exists()
-
-    @property
-    def supporters(self):
-        return [supporter.person for supporter in self.supporter.all()]
-
-    def add_submitter(self, person):
-        MotionSubmitter.objects.create(motion=self, person=person)
-
-    def clear_submitters(self):
-        MotionSubmitter.objects.filter(motion=self).delete()
+    def is_submitter(self, user):
+        """
+        Returns True if user is a submitter of this motion, else False.
+        """
+        return user in self.submitters.all()
 
-    def is_supporter(self, person):
+    def is_supporter(self, user):
         """
-        Return True, if person is a supporter of this motion. Else: False.
+        Returns True if user is a supporter of this motion, else False.
         """
-        return self.supporter.filter(person=person.pk).exists()
+        return user in self.supporters.all()
 
-    def support(self, person):
+    def support(self, user):
         """
-        Add 'person' as a supporter of this motion.
+        Add 'user' as a supporter of this motion.
         """
         if self.state.allow_support:
-            if not self.is_supporter(person):
-                MotionSupporter(motion=self, person=person).save()
+            if not self.is_supporter(user):
```

Comment:
I think this line can be removed. If not (please test it) it can be merged with line 408